### PR TITLE
Update FR lang to latest changes

### PR DESCRIPTION
**Pull request by FelixINX**

### CODE REVIEW

File: lang/fr/lang.php [hunk 1]
```diff
@@ -1,9 +1,8 @@
-<?php
-
-return [
+<?php return [
     'plugin' => [
         'name' => 'Traductions',
         'description' => 'Permet de créer des sites Internet multilingues',
+	'tab' => 'Traduction',
```
Comment (CptMeatball) [inline lang/fr/lang.php]: Indentation issue.

File: lang/fr/lang.php [hunk 2]
```diff
@@ -18,28 +17,37 @@
         'select_label' => 'Sélectionner une langue',
         'default_suffix' => 'défaut',
         'unset_default' => '":locale" est déjà la langue par défaut et ne peut être désactivée',
+	'delete_default' => '":locale" est la valeur par défaut et ne peut pas être supprimé.',
```
Comment (CptMeatball) [inline lang/fr/lang.php]: Identation issue.

File: lang/fr/lang.php [hunk 3]
```diff
@@ -18,28 +17,37 @@
...
         'name' => 'Nom',
         'code' => 'Code',
         'is_default' => 'Défaut',
         'is_default_help' => 'La langue par défaut représente le contenu avant la traduction.',
-        'is_enabled' => 'Activer',
+        'is_enabled' => 'Activé',
         'is_enabled_help' => 'Les langues désactivées ne seront plus disponibles sur le site.',
         'not_available_help' => 'Aucune autre langue n\'est définie.',
         'hint_locales' => 'Vous pouvez ajouter de nouvelles langues et traduire les messages du site. La langue par défaut est celle utilisée pour les contenus avant toute traduction.',
+	'reorder_title' => 'Réorganiser les langues',
```
Comment (CptMeatball) [inline lang/fr/lang.php]: Indentation issue.

File: lang/fr/lang.php [hunk 4]
```diff
@@ -18,28 +17,37 @@
...
+        'sort_order' => 'Ordre de tri',
     ],
     'messages' => [
         'title' => 'Traduction des Messages',
-		'description' => 'Mettre à jour Messages',
+	'description' => 'Mettre à jour Messages',
```
Comment (CptMeatball) [inline lang/fr/lang.php]: Indentation issue.

**Comment by LukeTowers** [comment]
@FelixINX please implement the suggestions made by @CptMeatball

**Comment by FelixINX** [comment]
@LukeTowers @CptMeatball This is weird but I don't see the indentation issue when I try to edit the file.

See here :

**Comment by LukeTowers** [comment]
@FelixINX check that all indentation is done with four spaces, not any tabs.

**Comment by FelixINX** [comment]
@LukeTowers should do it. Sorry about this!

**Comment by LukeTowers** [comment]
Thanks for your work @FelixINX!

**Comment by FelixINX** [comment]
In return I have a Hacktoberfest tshirt! 😉

**Pull request merged**